spec/requests/api/v1/github_webhook/filter_spec.rb

```diff
@@ -200,3 +180,15 @@
     end
   end
 end
+
+def expect_notification(text: message)
```

Comment:
Just curious, what is the colon meanning in a function parameter => `text: meesage`, Which is the reason why you don't use just the variable `message`?

Reply:
you can pass parameters in the form of expect_notification(text: 'something'), so it's more clear what parameter are you sending to the method, in this case :text

text: message means that the default value is message, in this case is a variable.

Reply:
ok, I thought that the default values where defined like this: `def expect_notification(text = message)`

Reply:
It's a ruby 2.0 thing. Keyword arguments https://thoughtbot.com/blog/ruby-2-keyword-arguments